lib/policyEvaluator/RequestContext.js

```diff
@@ -123,11 +123,18 @@ function _findAction(service, method) {
     if (service === 'sts') {
         return _actionMapSTS[method];
     }
+    if (service === 'metadata') {
+        const actions = {
```

Comment:
minor: for consistency why not having a top-level `_actionMapMetadata` object instead

Reply:
No reason. Originally I intended to have one action mapped to all metadata requests but decided later to split up admin and bucketd routes. I can do it.